core/src/components/item-divider/item-divider.ios.vars.scss

```diff
@@ -8,7 +8,7 @@
 $item-divider-ios-min-height:             28px !default;
 
 /// @prop - Font size of the item
-$item-divider-ios-font-size:              17px !default;
+$item-divider-ios-font-size:              1.0625rem !default;
```

Comment:
The default value this is set at (`17px`) is actually wrong when compared against native. It should be `15px` (or `0.9375rem`) at 100%. Something to keep in mind if you are comparing this against native.

Updating the default value is outside the scope of this feature & we have an open ticket to investigate changes for iOS 17 so we could update the font size as part of that.